### ISSUE TITLE
Electron:SelectBox: Regression - aria labeling <select> fails, Electron assigns incorrect role

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#60929]
Main aria-label for select element added in Commit 6babef94104b27b252929324aa3c2d69cd38e9f2
July 5 with Electron 1.72 stopped working at some point presumably with one of Electron 2.x
versions.

- Works again with Exploration version with Electron 3
- Need to identify exact problem version
- May need intermediary workaround until 3.x introduction

Related: #42476

**Comment by cleidigh** [comment]
@bpasero 
@isidorn 
@chrmarti 
In case any of you run into similar ARIA/Electron issues, FYI my findings:

Electron 1.x 2.x and 3 all assign different and consequential roles for a ```<select>``` with no explicitly assigned role (w3c default expectation)

1.x  select => assigned to role=listitem - explicit labeling works
2.x select => role=menuitem - prevents proper accessibility naming
3 (tested within Exploration) select => role=combobox - naming appears corrected

Workaround: Explicitly set role=combobox  - initial tests seem to correct problem

I imagine there are a variety of similar issues perhaps with other elements.  Please Note for any other interested parties.  I will try to check other scenarios like the checkbox issue for breakpoints.

I will PR this but should I keep open since it's a workaround?

**Comment by isidorn** [comment]
@cleidigh thanks for the explanation. Yeah we can keep this open so we are aware of the workaround.
Since we should remember to revert the workaround once we merge electron 3.0 in November / December

**Comment by cleidigh** [comment]
Have to use workaround for native select / Voiceover
PR: #62504

**Comment by isidorn** [comment]
This should be fixed with latest insiders since we use electron 3.0

**Comment by bpasero** [comment]
How to verify? I think I am seeing the same output in NVDA when focus is on the debug launch configurations dropdown.

**Comment by bpasero** [comment]
@cleidigh should your change be reverted now that we are on Electron 3? I see this comment (twice in our code):

```
// Workaround for Electron 2.x
// Native select should not require explicit role attribute, however, Electron 2.x
// incorrectly exposes select as menuItem which interferes with labeling and results
// in the unlabeled not been read.  Electron 3 appears to fix.
this.selectElement.setAttribute('role', 'combobox');
```

**Comment by cleidigh** [comment]
@bpasero 
I believe so, but I've been pretty surprised before by odd behavior in this department.  I will test Windows and OS X  with the latest electron build to make sure and post a subsequent PR.

**Comment by bpasero** [comment]
Thanks
